doc/configuration.rst

```diff
@@ -363,6 +363,8 @@ Arguments:
   - coil (int): index of the coil e.g. 3
   - invert (bool, default=False): whether the logic level is inverted
     (active-low)
+  - write_multiple (bool, default=False): optional, whether perform write
+    using multiple coils method
```

Comment:
```suggestion
  - write_multiple (bool, default=False): whether to perform write
    using "write multiple coils" method instead of "write single coil"
```

Reply:
Maybe rename the argument `write_multiple` -> `write_multiple_coils`? I think that makes it even more clear.